src/components/Card/Card.js

```diff
@@ -136,8 +136,8 @@ const Card = props => (
           },
           fontWeight: 'normal',
           width: '100%',
-          '-webkit-line-clamp': '2',
-          '-webkit-box-orient': 'vertical',
+          webkitLineClamp: '2',
```

Comment:
That's one doesn't work ... it will produce 'webkit-line-clamp' instead of '-webkit-line-clamp'

Reply:
@eouin Thanks, the first letter should have been capital. I was getting the below 
<img width="1098" alt="Screenshot 2019-03-19 at 09 55 23" src="https://user-images.githubusercontent.com/44162302/54596789-47cb4b80-4a2d-11e9-882c-988943880e2b.png">